capa/render/default.py

```diff
@@ -148,7 +152,60 @@ def render_attack(doc: rd.ResultDocument, ostream: StringIO):
     if rows:
         ostream.write(
             tabulate.tabulate(
-                rows, headers=[width("ATT&CK Tactic", 20), width("ATT&CK Technique", 80)], tablefmt="mixed_grid"
+                rows,
+                headers=[width("ATT&CK Tactic", 20), width("ATT&CK Technique", 80)],
+                tablefmt="mixed_grid",
+            )
+        )
+        ostream.write("\n")
+
+
+def render_maec(doc: rd.ResultDocument, ostream: StringIO):
+    """
+    example::
+
+        +--------------------------+-----------------------------------------------------------+
+        | MAEC Schema              | MAEC Value                                                |
+        |--------------------------+-----------------------------------------------------------|
+        | analysis-conclusion      | malicious                                                 |
+        |--------------------------+-----------------------------------------------------------|
+        | malware-family           | PlugX                                                     |
+        |--------------------------+-----------------------------------------------------------|
+        | malware-category         | downloader                                                |
+        |                          | launcher                                                  |
+        +--------------------------+-----------------------------------------------------------+
+    """
+    maec_schema: dict[str, set[str]] = {
+        "analysis-conclusion": set(),
+        "analysis-conclusion-ov": set(),
+        "malware-family": set(),
+        "malware-category": set(),
+        "malware-category-ov": set(),
+    }
+
+    for rule in rutils.maec_rules(doc):
+        if rule.meta.maec.analysis_conclusion:
```

Comment:
create an inner loop here using `for k,v in maec_schema...`?

Reply:
then append directly to rows?

Reply:
I *think* we can rely on the order of dict enumeration since 3.7. In that version, Python started guaranteeing that enumeration order is derived from insertion order. I'm just not sure how that works with dict literals.

Reply:
ah looks guaranteed too: https://stackoverflow.com/a/63876094